src/models.js

```diff
@@ -5483,6 +5524,10 @@ const MODEL_FOR_AUDIO_CLASSIFICATION_MAPPING_NAMES = new Map([
     ['audio-spectrogram-transformer', ['ASTForAudioClassification', ASTForAudioClassification]],
 ]);
 
+const MODEL_FOR_XVECTOR_MAPPING_NAMES = new Map([
```

Comment:
```suggestion
const MODEL_FOR_AUDIO_XVECTOR_MAPPING_NAMES = new Map([
```

Names are from [this file](https://github.com/huggingface/transformers/blob/d3a4b4754440041b20247365f33fb8e44c6d9caf/src/transformers/models/auto/modeling_auto.py#L1143).

Reply:
This is really helpful!